lib/xcodeproj/workspace/file_reference.rb

```diff
@@ -73,6 +73,8 @@ def absolute_path(workspace_dir_path)
           File.expand_path(File.join(workspace_dir_path, path))
         when 'container'
           File.expand_path(File.join(workspace_dir_path, path))
+        when 'self'
```

Comment:
nit: we can potentially combine all of these cases since they all return the same.

Reply:
Good point, I was trying to change the minimum. Should I combine the first three cases as a multi-value case? I tried this locally and tests pass

``` def absolute_path(workspace_dir_path)
        workspace_dir_path = workspace_dir_path.to_s
        case type
        when 'group', 'container', 'self'
          File.expand_path(File.join(workspace_dir_path, path))
        when 'absolute'
          File.expand_path(path)
        when 'developer'
          raise "Developer workspace file reference type is not yet supported (#{path})"
        else
          raise "Unsupported workspace file reference type `#{type}`"
        end
      end
```

Reply:
yes!